travis-build.sh

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
 
-if [ -z "$STACK_YAML" ]; then
-    cabal new-build all --enable-tests
-    cabal new-test all --enable-tests
-    # git fetch origin master:origin/master && git rebase origin/master --exec "cabal new-build all --enable-tests"
-else
-    # install stack
-    curl -sSL https://get.haskellstack.org/ | sh
-
-    # build project with stack
-    stack --version
-    stack build --system-ghc --test
-    # git fetch origin master:origin/master && git rebase origin/master --exec "stack build --system-ghc --test"
-fi
+# Cabal
+
+cabal new-build all --enable-tests
+cabal new-test all --enable-tests
+# git fetch origin master:origin/master && git rebase origin/master --exec "cabal new-build all --enable-tests"
+
+# Stack
+
+export STACK_YAML="stack.${TRAVIS_HASKELL_VERSION}.yaml"
```

Comment:
this can be set directly using `stack build --system-ghc --test --stack-yaml "stack.${TRAVIS_HASKELL_VERSION}.yaml"`.

Reply:
Good to know! But I'll leave it like this since I'll be using the setting twice.